modules/ROOT/pages/siebel/siebel-connector.adoc

```diff
@@ -156,13 +156,26 @@ Limit the number of objects to retrieve through DataSense to a few objects using
 
 == Log Requests and Responses [[log-requests-responses]]
 
-To log requests and responses when using the connector, configure the `log4j.xml` configuration file for the Mule app as follows:
+To log requests and responses when using the connector, add the following line to the `Loggers` element of the `log4j2.xml` configuration file for the Mule app:
 
 [source,xml,linenums]
 ----
 <AsyncLogger name="org.mule.modules.siebel.internal.service"
    level="DEBUG"/>
 ----
+
+You can view the app log  as follows:
```

Comment:
Not sure what you mean by "app log" -- please explain in the text.

Reply:
This is wording from the Mule documentation.